apps/speed.c

```diff
@@ -1466,6 +1486,21 @@ int speed_main(int argc, char **argv)
             if (!opt_int(opt_arg(), &primes))
                 goto end;
             break;
+        case OPT_SECONDS:
+            seconds.sym = seconds.rsa = seconds.dsa = seconds.ecdsa
+                        = seconds.ecdh = atoi(opt_arg());
+            break;
+        case OPT_BYTES:
+            lengths_single = atoi(opt_arg());
+            if (lengths_single >= BUFSIZE) {
+                BIO_printf(bio_err,
+                           "%s: %d bytes exceeds max\n",
```

Comment:
May I suggest to write even the max value? I mean if you expose this option to user, then it would be only appropriate to inform about the limit without having to refer to source code. I suppose one should print BUFSIZE-1. [Just in case, no, I don't know why it's defined with `+1`.]

On size note, multiblock_speed allocates own buffer and is free from this limitation.

Reply:
Oh! While you are at it, just to spare a separate merge request, may I ask you to remove `(int)` casts in `app_malloc((int)BUFSIZE ...)`. [Just in case, no, I don't know why they were added back, but I'm sure it's redundant]. If you don't feel like it, it's fine, I'll submit separate request myself.

Reply:
ok, got completely rid of BUFSIZE and SIZE_NUM limitations now. int casts were removed.

Reply:
Cool! Thanks!